api/tools/README.md

```diff
@@ -0,0 +1,17 @@
+# All of Us Workbench TOOLS
+
+## Load Data Dictionary
```

Comment:
Thanks for documenting this. What needs to happen to get this in prod? Also, can we document this in the [CDR Playbooks](https://docs.google.com/document/d/17YVCqu4BhVV8ai3OutHx8u0m6V5NkdM5H9kLNxTPDdo/edit#heading=h.676040liklzd) since this is likely to happen with the publish of a new CDR in the future. You can probably start a section for this under **Enabling New CDRs**

Reply:
Will add the entry in CDR playbooks.
the command to execute LoadDataDictionary is part of deploy command, so we will not have to do anything special, the weekly release should take care of this . I'll add that as a note here